src/napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -107,7 +108,10 @@ def __init__(self, layer: Layer) -> None:
             False,
             self.PAN_ZOOM_ACTION_NAME,
             extra_tooltip_text=trans._(
-                '\n(or hold Space)\n(hold Shift to pan in 3D)'
+                '\n(or hold Space)'
+                '\n(hold {control} to zoom-in using region of interest)'
+                '\n(hold Shift to pan in 3D)',
```

Comment:
```suggestion
                '\n(or hold Space)'
                '\n(hold Shift to pan in 3D)',
                '\n(hold {control} to zoom via ROI selection)'
```
I'm trying to think of something to make it more clear that this isn't using any already present ROI on the interface, such as a shapes layer, but is rather done using an action. 
I also would prefer this added to the end of the tooltip, but that's just a nit